### PR TITLE
Add Contributing.md file

### PR DESCRIPTION
**Pull request by smlambert**
Add missing Contributing.md file

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Contributing to aqa-tests
+
+Thanks for your interest in this project.
+
+## Project description
+
+aqa-tests is the central project for AQAvit and contains the definition files for the AQA test suites
```
Comment (sophia-guo) [inline Contributing.md:7]: Not sure if  capitalize `aqa-tests` would be better.

**Review by sophia-guo — APPROVED** [review]

File: Contributing.md (hunk 1 above)
Comment (smlambert) [reply Contributing.md:7]: I am matching it to the github repo name which is all lowercase (and how we did the aqa-test-tools Contributing.md file also)

**Review by llxia — APPROVED** [review]
LGTM

**Comment by llxia** [comment]
@smlambert ECA failed with the error `Anonymous (smlamber@****.ibm) is not covered by the necessary legal agreements in order to proceed.`

**Comment by smlambert** [comment]
Fascinating... I guess this is because I used the web interface to add this file, rather than signing off from IDE, will attempt to update.

**Comment by gdams** [comment]
> Fascinating... I guess this is because I used the web interface to add this file, rather than signing off from IDE, will attempt to update.

@smlambert next time you use the web UI you'll notice a drop-down where you can select which email you want to create the commit with. Make sure to select the one that's linked to your EF account

**Comment by smlambert** [comment]
I am not seeing a drop-down?

Screen Shot 2021-06-03 at 11 45 42 AM

**Comment by smlambert** [comment]
Closing this in favour of https://github.com/adoptium/aqa-tests/pull/2644

**Pull request closed without merging**